platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/config/VirtualMapConfig.java

```diff
@@ -82,6 +89,8 @@ public record VirtualMapConfig(
         @Min(0) @Max(100) @ConfigProperty(defaultValue = "50.0")
                 double percentHashThreads, // FUTURE WORK: We need to add min/max support for double values
         @Min(-1) @ConfigProperty(defaultValue = "-1") int numHashThreads,
+        @Min(1) @Max(64) @ConfigProperty(defaultValue = "6") int virtualHasherChunkHeight,
+        @Min(0) @ConfigProperty(defaultValue = "500000") int reconnectFlushInterval,
```

Comment:
It's best to include the units of the value in the name of a variable. E.g. `reconnectFlushIntervalInNodes'. This is especially important for this specific parameter because the noun "interval" is usually reserved for temporal units, so the current name of this parameter is very confusing.

Reply:
There is a similar config `flushInterval` (which should be deprecated and eventually removed), which is also not in any time units, but in rounds. I added a JavaDoc comment for this new param, please, check if it's confusing and/or not enough

Reply:
I have seen the javadoc. My comment is specifically about the name of the parameter(s).

Reply:
I'm not sure if this is actually resolved, but oh well...